templates/csharp/app-reactiveui/ViewModels/MainWindowViewModel.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using ReactiveUI.Fody.Helpers;
+
+namespace AvaloniaAppTemplate.ViewModels
+{
+    public class MainWindowViewModel : ViewModelBase
+    {
+        [Reactive] public string Greeting { get; set; } = "Welcome to Avalonia!";
```

Comment:
Probably worth showing off bindings to `IObservable`s, `ReactiveCommand`s, and [ReactiveUI.Validation](https://github.com/reactiveui/reactiveui.validation) usage here? These patterns are quite common in Avalonia.ReactiveUI programming world so probably worth showing these features to the newcomers.

Reply:
Great idea, I wanted to do this but was afraid to complicate the example. Now I have an idea how to do it 😏

Reply:
Added example work with validation and IObservable